packages/superset-ui-connection/test/SupersetClientClass.test.ts

```diff
@@ -1,5 +1,6 @@
 import fetchMock from 'fetch-mock';
 
+import { BigNumber } from 'bignumber.js';
```

Comment:
do you need the `.js` here? it shoudl auto-resolve without it.

Reply:
Good point, I'll remove it - it was probably added by WebStorm.

Reply:
It looks like `bignumber.js` is the actual name of the module, so I can't remove the .js.

Reply:
ohhhh, okay. that's ... interesting :)